src/BrowserDetector/BrowserDetector.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import some = require('lodash/some');
+import { props, t, ReactChildren } from '../utils';
+import * as bowser from 'bowser';
+
+// FIXME: this is hardcoded and there's no guarantee it matches with bowser's types
+const browsers = [
```

Comment:
these are values.

Also, no need for `@types/bowser`, since https://github.com/lancedikson/bowser/blob/master/typings.d.ts